core/polypatch.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * @license
+ * Visual Blocks Editor
+ *
+ * Copyright 2025 Nitro-Bolt.
```

Comment:
```suggestion
 * Copyright 2025 NitroBolt.
```

Just because the organization name is cursed with `Nitro-Bolt`, doesn't mean we should impose it everywhere 🥲

We still truly are NitroBolt from the heart ❤️